extensions/widgets/gradientrampeditor/gradientrampeditor.lcb

```diff
@@ -285,6 +285,10 @@ public handler OnMouseDoubleUp() returns nothing
    gradientStopsChanged()
 end handler
 
+public handler onMouseCancel() returns nothing
+   put false into mIsDraggingStop
```

Comment:
The mouseCancel handler should function in the same way as mouseUp, by making sure that we redraw and send the gradient stop changed message. 

```
put xCoordToStopNumber(the x of the mouse position) into mCurrentlySelectedStopIndex
redraw all
gradientStopsChanged()
```

It might be worth refactoring the common code in mouseUp and mouseCancel into a new handler.